Cargo.toml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Just curious about this, why this is not inside the `contracts` folder ? Initially i put it there because the root is shared with the `app` folder, which contains the UI code.

Reply:
I don't think this is a major issue for now. There is no code just a Cargo.toml.